source/chapters/1/users.md

```diff
@@ -63,55 +64,56 @@ Another thing you may notice is how `emails` key is structured:
 ]
 ```
 
-This may seem a bit complicated, but they decided to stick with this, maybe because they wanted to satisfy easily the people
-who want multiple email addresses on their account. To get the email of your user, you would have to do:
+This may seem to be redundant and complicated, but the Meteor developers decided to stick with this, maybe because they wanted to cater to the people
+that want multiple email addresses linked to their account. To get the email of your user, you would have to use the following snippet:
 ```js
 Meteor.user().emails[0].address
 ```
 
-But don't worry about this now, when we'll learn how to make this easy, so you won't have to type this everywhere you need an user's email.
-
-You think '12345' is not a very secure password, and you are correct, let's change it:
+But you should not worry about this right now, because, by the end of this tutorial we'll learn how to easily implement this in any application, 
+and simplify the steps you'll need to take in order to use it, so you won't have to repeat yourself whenever you will need a user's email.
+You might think that '12345' is not a very secure password! Since you are correct, let's go ahead and change it:
 
 ```js
-Accounts.changePassword('12345', 'My1337L333Tpasswurt%', function (err) {
+Accounts.changePassword('12345', 'My1337L333Tpassword%', function (err) {
     if (!err) {
-        console.log('Change password was a success!')
+        console.log('Changing the password was a success!')
     } else {
         console.log(err);
     }
 })
 ```
 
-Very nice, now let's try a logout:
+Very nice! Now, since the password wasn't abiding to the basic guidelines that the user should respect when choosing his/her password, we made them change it.
+ Now let's get the user to try out his/her new shiny password by logging them out, in order to get them to log back in with the new password:
 
 ```js
 Meteor.logout(function (err) {
     if (!err) {
-        console.log('Logout was a success!')
+        console.log('The user was logged out successfully !')
     } else {
         console.log(err);
     }
 });
-// now Meteor.user() and Meteor.userId() will be null
 ```
+Now, if you check their values, *Meteor.user()* and *Meteor.userId()* will be null, because there is no user that's currently logged in.
 
-Next time you login, you'll login with your new password.
-
-Btw, the callbacks we used in `loginWithPassword`, `changePassword` and `logout` are optional, you can simply not use it.
-
-But wait, your new password is so complex, you already forgot it.
+The [callbacks](https://en.wikipedia.org/wiki/Callback_(computer_programming)) we used previously in `loginWithPassword`, 
```

Comment:
Not wikipedia or something very general, let's get into specifics, I'd say: https://developer.mozilla.org/en-US/docs/Glossary/Callback_function or something similar